tuning/config/configs.py

```diff
@@ -30,7 +30,6 @@ class DataArguments:
 
 @dataclass
 class TrainingArguments(transformers.TrainingArguments):
-    peft_method: str = "lora"  # None, pt
```

Comment:
I got rid of this. users can pass the relevant peft config object in train() that will be passed directly to the trainer